CHANGES

```diff
@@ -4,6 +4,8 @@ v2016.1-dev 2016-07-04
  - Start v2016.1
  - Fix https://github.com/KhronosGroup/SPIRV-Tools/issues/261
    Turn off ClipDistance and CullDistance capability checks for Vulkan.
+ - Validator is incomplete
+   - Checks ID use block is dominated by definition block
```

Comment:
v2016.2-dev has begun.  Please move this up.  (And thanks for being pro-active about updating CHANGES!)